<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaveWindow</class>
 <widget class="QWidget" name="CaveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>639</x>
     <y>40</y>
     <width>121</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="GenerateCave">
      <property name="text">
       <string>Generate Cave</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="LoadFile">
      <property name="text">
       <string>Load from file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="Chance">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>191</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="Birth">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>222</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="Death">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>253</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="Rows">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>129</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="Cols">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>624</x>
     <y>129</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cols</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>628</x>
     <y>222</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Birth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>621</x>
     <y>253</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Death</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>191</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextStep">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>290</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Auto">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>320</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Time">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>9999994</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="GenerateMaze">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>510</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Maze</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLine">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>560</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
